pkg/agent/sysadvisor/types/types.go

```diff
@@ -113,16 +112,16 @@ type PoolInfo struct {
 
 // RegionInfo contains region information generated by sysadvisor resource advisor
 type RegionInfo struct {
-	RegionType   QoSRegionType
-	BindingNumas machine.CPUSet
+	RegionType   QoSRegionType  `json:"region_type"`
+	BindingNumas machine.CPUSet `json:"binding_numas"`
 
-	HeadroomPolicyTopPriority CPUHeadroomPolicyName
-	HeadroomPolicyInUse       CPUHeadroomPolicyName
-	Headroom                  resource.Quantity
+	HeadroomPolicyTopPriority CPUHeadroomPolicyName `json:"headroom_policy_top_priority"`
+	HeadroomPolicyInUse       CPUHeadroomPolicyName `json:"headroom_policy_in_use"`
+	Headroom                  string                `json:"headroom"`
```

Comment:
Changing RegionInfo member type will affect reference in adapter, so also need to modify in adapter. How about use float64 for headroom since it is actually a value.

Reply:
done